src/java/org/apache/cassandra/index/sai/disk/v1/MemtableIndexWriter.java

```diff
@@ -146,6 +146,7 @@ public void complete(Stopwatch stopwatch) throws IOException
 
     private long flush(DecoratedKey minKey, DecoratedKey maxKey, AbstractType<?> termComparator, MemtableTermsIterator terms, int maxSegmentRowId) throws IOException
     {
+        long cellCount;
```

Comment:
Why do we need to use different words for the same thing. Now you introduced yet another word: cell.
What was wrong with 'postings'?

Reply:
I tried to explain it in this comment: https://github.com/datastax/cassandra/pull/1695#discussion_r2056836437
Can you explain what I am missing there?